### ISSUE TITLE
training on a new language

### ISSUE THREAD

**Issue by mohammed-bahumaish** [microsoft/VibeVoice#115]
Hi @YaoyaoChang 
Is there already a script for pretraining on a new language for the streaming model, if not, what's required to make?

**Comment by YaoyaoChang** [comment]
We do not plan to allow fine-tuning of the streaming model, due to potential DeepFake risks.
Which new languages would you like?

**Comment by utkarshshukla2912** [comment]
hindi.

**Comment by mohammed-bahumaish** [comment]
> We do not plan to allow fine-tuning of the streaming model, due to potential DeepFake risks. Which new languages would you like?

Arabic

**Comment by giandiego** [comment]
Spanish

**Comment by ziom6270** [comment]
Polish.

I tried every model, awesome. Please train Polish models or git the way to do that, it would bee awesome for story telling.

**Comment by Sheldonimo** [comment]
Spanish

**Comment by wenhui0924** [comment]
Thanks for your feedback! We did some initial tests using English speakers, and it appears that the model can produce German, Spanish, Portuguese, Japanese, and Arabic to some extent. However, we haven’t conducted enough training or thorough evaluation for these languages, so we can’t guarantee the quality or stability of the results at this stage.

The actual performance may vary, and we recommend testing your own use cases to see how well it works for you. You can also test other languages and provide us with some feedback, because we don’t understand many of them and aren’t sure how well they actually perform. We also plan to provide more multilingual speaker embeddings in the future to improve cross-lingual performance.

Thanks again for the suggestion and for trying the model!

**Comment by haseebsultankhan** [comment]
Urdu language please

**Comment by ziom6270** [comment]
@wenhui0924 

I conducted several tests on the base text, which was generated with consideration for the specifics of the Polish language. In the attachments, I am including a report consisting of partial logs from the generation process, the source text, and the generated files.

As for my reflection on the model, I did not notice the occurrence of so-called “hallucinations,” and the generation is very fast. Regarding audio specifications, the best performance is achieved with the settings cfg1.5/inference5 or cfg1.7/inference8. The model has a distinctive English accent, reminiscent of a person with Polish roots but raised in an English-speaking country—something like a child emigrating permanently to the USA at the age of seven and then returning to Poland after 30 years (never using the Polish language again in the meantime).

I suspect that if the model is further trained with a Polish audio dataset (flac/wav-320kbps) along with detailed transcription, it will be able to pronounce all the nuances of the language with 100% accuracy. At present, I must admit that such complex combinations of letters like rz/sz/cz/ś/ć/ź/ń/ó/ą/ę/ż remain challenging.

Personally, I use the XTTSv2 2.0.3 model, but I must say that VibeVoice-Realtime-0.5B is very promising, especially considering the absence of hallucinations. The prosody and the spacing between commas and periods are at a very high level.

I look forward to your possible corrections and improvements :)

[1p_vibevoice.txt](https://github.com/user-attachments/files/23959595/1p_vibevoice.txt)
[1p_vibevoice_generated_cfg1.1_inference3.wav](https://github.com/user-attachments/files/23959600/1p_vibevoice_generated_cfg1.1_inference3.wav)
[1p_vibevoice_generated_cfg1.5_inference5.wav](https://github.com/user-attachments/files/23959601/1p_vibevoice_generated_cfg1.5_inference5.wav)
[1p_vibevoice_generated_cfg1.7_inference8.wav](https://github.com/user-attachments/files/23959602/1p_vibevoice_generated_cfg1.7_inference8.wav)
[1p_vibevoice_generated_cfg3_inference5.wav](https://github.com/user-attachments/files/23959598/1p_vibevoice_generated_cfg3_inference5.wav)
[1p_vibevoice_generated_cfg3_inference20.wav](https://github.com/user-attachments/files/23959596/1p_vibevoice_generated_cfg3_inference20.wav)
[2p_vibevoice.txt](https://github.com/user-attachments/files/23959603/2p_vibevoice.txt)
[2p_vibevoice_generated_cfg1.5_inference5.wav](https://github.com/user-attachments/files/23959599/2p_vibevoice_generated_cfg1.5_inference5.wav)
[raports.txt](https://github.com/user-attachments/files/23959597/raports.txt)

**Comment by dingausmwald** [comment]
German please

**Comment by onedge** [comment]
Korean please

**Comment by bioluminesceme** [comment]
Dutch please

**Comment by jozadaquebatista** [comment]
Brazillian Portuguese, please

**Comment by duyquangnvx** [comment]
Thanks for the update! Vietnamese support would be extremely valuable

**Comment by sefaalper** [comment]
Turkish support please

**Comment by wwfcnu** [comment]
chinese support please

**Comment by devops724** [comment]
Persian

**Comment by SwapniL-afk-now** [comment]
Bengali

**Comment by YaoyaoChang** [comment]
We introduce experimental speakers in 9 languages (DE, FR, IT, JP, KR, NL, PL, PT, ES) for exploration. Please pull the latest code to try them out.
Due to limited testing and our lack of native expertise in these languages, quality may vary across languages and speakers. Community feedback is highly appreciated.
We will continue to improve multi-lingual support in future releases.

**Comment by dingausmwald** [comment]
This is huge, thanks alot! Is it possible that you update the demo on huggingface so we can test the new languages?

**Comment by bioluminesceme** [comment]
## Dutch 

### Dutch - Male.
Very acceptable. Good enough for me.

Points of potential improvement, these things happened roughly once in 3+ minutes of streamed audio:

- a bit robotic at times
- at times the inflection slightly off
- static hiss around the 480 second mark (testing realtime with my PR fix that sends the text as a payload vs URL param)
- it does seem to have hint of a regional accent at times (I heard a Limburg soft g/ some Amsterdam tonality which does not fit with the rest of the voice)


### Dutch - Woman.

This one is not good. It is Flemish (so Belgium BE, not the Netherlands NL)
It also inserts frequent uh's/sighs/inhalation sounds.

**Comment by siberianwm** [comment]
Hi VibeVoice team! 👋

First of all, huge kudos on the release! The architecture and the realtime capabilities are truly impressive, especially the low latency streaming.

I noticed in the config (`vibevoice/configs/qwen2.5_1.5b_64k.json`) that the base model is **Qwen 2.5 1.5B**. Since Qwen 2.5 inherently has excellent multilingual capabilities (including strong support for **Russian**), I was wondering about the roadmap for extending VibeVoice to support it.

Currently, the README states support is limited to English and Chinese (with some experimental languages added recently).

**My questions:**
1. Are there any official plans to release an acoustic tokenizer/head fine-tuned for Russian in the near future?
2. If not, would you be open to providing a training recipe/script so the community can fine-tune the acoustic part for other languages utilizing the underlying Qwen capabilities?

This would be a game-changer for building fully local realtime assistants in Russian, as there are currently no open-source streaming alternatives with this level of expressiveness.

Thanks again for the great work!

**Comment by ductaiii** [comment]
> We do not plan to allow fine-tuning of the streaming model, due to potential DeepFake risks. Which new languages would you like?

i think vietnamese

**Comment by ziom6270** [comment]
Exactly as **_syberianwm_** wrote, it's not so much a cloning/fine-tuning script, but a script that further trains the linguistic range of a given language with the default base voice(s). It would be great to be able to "supercharge" the model with our own dataset not for voice-cloning, but for its understanding power.

**Comment by sd983527** [comment]
> Exactly as **_syberianwm_** wrote, it's not so much a cloning/fine-tuning script, but a script that further trains the linguistic range of a given language with the default base voice(s). It would be great to be able to "supercharge" the model with our own dataset not for voice-cloning, but for its understanding power.

Acoustic tokenizer is must have in the fine-tuning process, so that it would not be possible that the model being finetuned but still safe. Maybe you could provide links for public datasets on the languages you may want to support and ask the vibevoice team to finetune with that data.

**Comment by ralodo93** [comment]
Spanish from Spain

**Comment by demateopablo** [comment]
Would be cool to have finally an realistic Argentinian Spanish support

**Comment by bakhtiyork** [comment]
> We do not plan to allow fine-tuning of the streaming model, due to potential DeepFake risks. Which new languages would you like?

Uzbek language, please

**Comment by cahya-wirawan** [comment]
> We do not plan to allow fine-tuning of the streaming model, due to potential DeepFake risks. Which new languages would you like?

Indonesian please.

**Comment by trungnk-png** [comment]
> We do not plan to allow fine-tuning of the streaming model, due to potential DeepFake risks. Which new languages would you like?

Vietnamese

**Comment by alarno** [comment]
Catalan please, how can we contribute to make it?

**Comment by MrMuhannadObeidat** [comment]
> We do not plan to allow fine-tuning of the streaming model, due to potential DeepFake risks. Which new languages would you like?

Arabic works to some extent but not at a level where it can be used in any application. It is on the other hand easy to train with relatively a 100K examples or so based on finetuning of the 1.5B parameter model. Opening this up for fine tuning is going to be be very useful. Why is deepfake a concern here but not in the original model or with all the other model out there?

**Comment by SnophiTheDeveloper** [comment]
Hello VibeVoice!
Is there any update on Turkish model? 

Thank you!

**Comment by RyanL-29** [comment]
> We do not plan to allow fine-tuning of the streaming model, due to potential DeepFake risks. Which new languages would you like?

I would really appreciate it if the streaming model support Cantonese / Yue.

**Comment by dwick90** [comment]
> We do not plan to allow fine-tuning of the streaming model, due to potential DeepFake risks. Which new languages would you like?

Hey,
Hungarian would be great.

**Comment by Mohcinimohamed** [comment]
I've been fine-tuning VibeVoice-ASR for Darija (Moroccan Arabic) using the official LoRA guide. 81 hours of speech data collected via a community app, trained with QLoRA on a single 16GB GPU.

Submitted PR #334 with a Darija toy dataset and training commands as a reference for others adding new languages. The LoRA finetuning scripts in finetuning-asr/ are language-agnostic — just prepare your data in the documented JSON format and run the same commands.

**Comment by sarathkumarm1207** [comment]
> We do not plan to allow fine-tuning of the streaming model, due to potential DeepFake risks. Which new languages would you like?

Tamil, We need a tamil language support
